lib/config/testdata_test.go

```diff
@@ -164,6 +164,23 @@ proxy_service:
   mysql_public_addr: mysql.example:3306
   mongo_listen_addr: webhost:27017
   mongo_public_addr: mongo.example:27017
+
+db_service:
+  enabled: yes
+  resources:
+    - labels:
+        "*": "*"
+  azure:
+    - subscriptions: ["sub1", "sub2"]
+      resource_groups: ["group1", "group2"]
+      types: ["postgres", "mysql"]
+      regions: ["eastus", "centralus"]
+      tags:
+        "a": "b"
+    - types: ["postgres", "mysql"]
+      regions: ["westus"]
+      tags:
+        "c": "d"
```

Comment:
Ignore this comment if you are planning to update `databaseAgentConfigurationTemplate` in future PRs.

Reply:
Not sure what you mean here?

Reply:
sorry. here is the link
https://github.com/gravitational/teleport/blob/5d9aa74375ae64cf2205993b09f80edd0bab16fe/lib/config/database.go#L36-L45

Reply:
Oh, yes that's in another PR: #15630